lib/etc/openturns.conf.in

```diff
@@ -867,6 +867,9 @@
   <SpectralGaussianProcess-StartingScaling   value_float="1.0e-13"         />
   <SpectralGaussianProcess-CholeskyCacheSize value_int="16384" />
 
+  <!-- OT::Gibbs parameters -->
+  <Gibbs-DefaultUpdatingMethod       value_int="0"       />
```

Comment:
better preserve the keys order wrt RM.cxx, it allows to spot diffs

Reply:
Done.